eng/nuget/Microsoft.NET.Runtime.Emscripten.Sdk/Microsoft.NET.Runtime.Emscripten.Sdk.props

```diff
@@ -6,7 +6,8 @@
   <ItemGroup>
     <File Include="$(ArtifactsDirToPackage)**" TargetPath="tools\%(RecursiveDir)" SkipPackageFileCheck="true" />
     <File Include="$(RepoRoot)eng\.emscripten" TargetPath="tools\emscripten" SkipPackageFileCheck="true" />
-    <File Include="$(RepoRoot)eng\emsdk_env.sh" TargetPath="tools" SkipPackageFileCheck="true" />
+    <File Condition="!$([MSBuild]::IsOSPlatform(Windows))" Include="$(RepoRoot)eng\emsdk_env.sh" TargetPath="tools" SkipPackageFileCheck="true" />
+    <File Condition="$([MSBuild]::IsOSPlatform(Windows))" Include="$(RepoRoot)eng\emsdk_env.cmd" TargetPath="tools" SkipPackageFileCheck="true" />
```

Comment:
actually thinking about it, shouldn't we always include both .sh and .cmd since the consuming user will need to decide?